### PR TITLE
Add support for i9505

### PR DESCRIPTION
**Pull request by rrooij**
Current status:

* Only works with exporting as recovery zip and flashing it with TWRP
* Currently boots Weston
* SSH access is possible on USB
* Touch screen does not work
* Screen has a red glare

### CODE REVIEW

File: aports/device/linux-samsung-i9505/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# Kernel config based on: arch/arm/configs/(CHANGEME!)
```
Comment (ollieparanoid) [inline aports/device/linux-samsung-i9505/APKBUILD]: Please replace the `(CHANGEME!)` part.

File: aports/device/linux-samsung-i9505/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+pkgname="linux-samsung-i9505"
+pkgver=3.4.112
+pkgrel=0
+pkgdesc="Samsung Samsung Galaxy S4 International kernel fork"
```
Comment (ollieparanoid) [inline aports/device/linux-samsung-i9505/APKBUILD]: Samsung once is enough there (I know it's automatically generated).

**Comment by rrooij** [comment]
Thanks for the comments. I will fix it ASAP.

**Comment by rrooij** [comment]
Done 👍

**Comment by rrooij** [comment]
Another thing: the demo `weston-simple-egl` runs fine. This means that OpenGL ES seems to be working fine.

**Pull request merged**